cpp/src/arrow/util/thread-pool.h

```diff
@@ -36,6 +36,11 @@
 
 namespace arrow {
 
+// Get the number of worker threads used by the process-global thread pool
+// for CPU-bound tasks.  This is an idealized number, the actual number
+// may lag a bit.
+ARROW_EXPORT size_t GetCPUThreadPoolCapacity();
```

Comment:
Use `int` here instead of `size_t`. See https://google.github.io/styleguide/cppguide.html#Integer_Types

Reply:
This style guide is a bit ambiguous. It says: """Of the C integer types, only int should be used. When appropriate, you are welcome to use standard types like size_t and ptrdiff_t."""

Does it consider size_t not to be a C integer type?